src/settings.rs

```diff
@@ -13,6 +13,43 @@ pub struct PaginationSetting {
     pub max_total_hits: usize,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct MinWordSizeForTypos {
+    pub one_typo: Option<u8>,
+    pub two_typos: Option<u8>,
+}
+
+impl Default for MinWordSizeForTypos {
+    fn default() -> Self {
+        MinWordSizeForTypos {
+            one_typo: Some(5),
+            two_typos: Some(9),
+        }
+    }
+}
```

Comment:
We shouldn't try to guess the default value of meilisearch by default, + we don't want to overwrite a previous call of the route.
Sending `None` is enough.

```suggestion
impl Default for MinWordSizeForTypos {
    fn default() -> Self {
        MinWordSizeForTypos {
            one_typo: None,
            two_typos: None,
        }
    }
}
```